main.go

```diff
@@ -160,15 +174,30 @@ func realMain(args []string) int {
 	return exit
 }
 
+func jsonLoader(path string) (gojsonschema.JSONLoader, error) {
+	buf, err := ioutil.ReadFile(path)
+	if err != nil {
+		return nil, err
+	}
+	switch filepath.Ext(path) {
+	case ".yml", ".yaml":
+		buf, err = yaml.YAMLToJSON(buf)
+	}
+	if err != nil {
+		return nil, err
+	}
+	return gojsonschema.NewBytesLoader(buf), nil
+}
+
 func printUsage() {
-	fmt.Fprintf(os.Stderr, `Usage: %s -s schema.json [options] document.json ...
+	fmt.Fprintf(os.Stderr, `Usage: %s -s schema.json|schema.yml [options] document.json|document.yml ...
```

Comment:
Could shorten this a tiny bit

```suggestion
	fmt.Fprintf(os.Stderr, `Usage: %s -s schema.(json|.yml) [options] document.(json|yml) ...
```

Reply:
yeah.. thanks for that.. I originally had [schema.yml|schema.json], but that didn't feel right because that implied both were optional..